static/js/filter.js

```diff
@@ -0,0 +1,37 @@
+import { render } from './partials/render.js'
+
+const input = document.querySelectorAll('fieldset input')
+
+function getFilterdata () {
```

Comment:
It would probably be nice to move this function to `data.js` so it can be called as data.get & data.filter or as data.getFiltered